saviSimulation/config.cfg

```diff
@@ -7,10 +7,13 @@ FRAME_RATE = 150
 MAX_SPEED = 0.15
 PERCEPTION_DISTANCE = 300
 WIFI_PERCEPTION_DISTANCE = 500
-NUMBER_UAS = 5
+NUMBER_UGV = 1
+NUMBER_UAV = 1
 RANDOM_SEED = 776
 REASONING_CYCLE_PERIOD = 5
-TREE_SIZE = 15
+TREE_SIZE = 50
 HOUSE_SIZE = 15
 THREAT_SIZE = 10
-UAS_SIZE = 26
\ No newline at end of file
+ANTENNA_SIZE = 26
+UGV_SIZE = 26
+UAV_SIZE = 26
```

Comment:
The wiki page on the configuration file will need to be updated to reflect the changes in this file.

Reply:
Ok, done.